packages/flame/lib/src/effects/effect.dart

```diff
@@ -45,7 +45,15 @@ abstract class Effect extends Component {
   bool removeOnFinish;
 
   /// Optional callback function to be invoked once the effect completes.
-  void Function()? onFinishCallback;
+  void Function()? onComplete;
+
+  @Deprecated('Use Effect.onComplete instead')
```

Comment:
This should state that it will be removed in v1.3.0

Reply:
Done. This is how the message will be displayed to end-users. 

> 'onFinishCallback' is deprecated and shouldn't be used. It will be removed in v1.3.0. Use Effect.onComplete instead.
> Try replacing the use of the deprecated member with the replacement.

Reply:
Great, good job with the PR!